src/lesson5/Main.java

```diff
@@ -0,0 +1,143 @@
+package lesson5;
+
+import java.util.concurrent.TimeUnit;
+
+public class Main {
+
+    protected static final int SIZE = 10000000;
+    protected static final int HALF = SIZE / 2;
+    protected static float[] arr1 = new float[SIZE];
+    protected static float[] arr2 = new float[SIZE];
+    protected static float[] arr2_1 = new float[HALF];
+    protected static float[] arr2_2 = new float[HALF];
+
+    public static void main(String[] args) throws Exception {
+
+        Thread thr1 = new Thread(new myCalculate1());
+        thr1.start();
+
+        Thread thr2 = new Thread(new myCalculate2());
+        thr2.start();
+    }
+
+    static class myCalculate1 extends Thread implements Runnable {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).